nilearn/plotting/img_plotting.py

```diff
@@ -366,34 +366,39 @@ def plot_anat(anat_img=MNI152TEMPLATE, cut_coords=None,
         black_bg: boolean, optional
             If True, the background of the image is set to be black. If
             you wish to save figures with a black background, you
-            will need to pass "facecolor='k', edgecolor='k'" to pylab's
-            savefig.
+            will need to pass "facecolor='k', edgecolor='k'" to plt.savefig.
         cmap: matplotlib colormap, optional
             The colormap for the anat
+        vmin: float
+            vmin passed to plt.imshow
```

Comment:
A possible alternative explanation for vmin/vmax would be:

```A lower/bigger bound filter that pushes any map value up/down smaller/bigger than vmin/max towards vmin/vmax.```

mention the analogy to np.clip()?

"Given an interval, values outside the interval are clipped to
the interval edges.  For example, if an interval of ``[0, 1]``
is specified, values smaller than 0 become 0, and values larger
than 1 become 1."